src/components/Search.js

```diff
@@ -1,9 +1,46 @@
-import React from "react";
+import React, { useState } from "react"
 
-function Search() {
-  function handleSubmit(e) {
-    e.preventDefault();
-    console.log("submitted");
+function Search({ listings, setListings, originalListings }) {
+  const [search, setSearch] = useState("")
+  const [sort, setSort] = useState("Sort by Location: ON")
+
+  const sortedListings = [...listings].sort((a, b) => {
+    const listingA = a.location
+    const listingB = b.location
+    if (listingA < listingB) {
+      return -1
+    }
+    if (listingA > listingB) {
+      return 1
+    }
+    return 0
+  })
```

Comment:
```suggestion
  const sortedListings = [...listings].sort((a, b) => a.location.localeCompare(b.location));
```

Reply:
I will look more into this method, much cleaner. Thanks!